cmd/copy.go

```diff
@@ -376,16 +376,16 @@ func (raw rawCopyCmdArgs) cook() (cookedCopyCmdArgs, error) {
 		}
 		// Disabling blob tier override, when copying block -> block blob or page -> page blob, blob tier will be kept,
 		// For s3 and file, only hot block blob tier is supported.
+		if cooked.fromTo == common.EFromTo.FileBlob() && cooked.blobType != common.EBlobType.None() {
+			return cooked, fmt.Errorf("file -> blob is not a supported direction while copying from service to service")
```

Comment:
>rf("file -> blob i [](start = 26, length = 18)

I think this message is wrong.  Shouldn't it be similar to the "custom blob type is not supported"  message that's a few lines below?  (I.e. currently its saying you can't do file ->  blob; but IIRC its supposed to say you can't do file-> blob  WITH a custom blob type

Reply:
Yes, you're right, that's a incorrect message.